tokio/src/io/util/copy.rs

```diff
@@ -94,7 +94,7 @@ impl CopyBuffer {
             feature = "time",
         ))]
         // Keep track of task budget
-        let coop = ready!(crate::runtime::coop::poll_proceed(cx));
+        let coop = ready!(crate::task::coop::poll_proceed(cx));
```

Comment:
You don't necessarily have to change the internal references - that makes it harder to review.